generative_ai/text_generation/gemini_describe_http_pdf_example.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+
+
+def generate_content() -> str:
+    # [START generativeaionvertexai_gemini_describe_http_pdf]
+    import vertexai
+    from vertexai.generative_models import GenerativeModel, Part
+
+    # TODO (developer): update project id
```

Comment:
nit: can you use this format: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/gemma2/gemma2_predict_gpu.py#L31-L32